docs/source/guide/measuring-resources-required.md

```diff
@@ -0,0 +1,105 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Measuring Resources Required
+
+Measuring the cost to run a quantum error mitigation (QEM) experiement would be ideally done in dollars.  This is difficult due to provider opacity, so cost should be measured by 
+
+- the number of circuits the technique uses
+- the number of additional gates
+
+This can be done by using mitiq's two stage application of ```mitiq.xyz.construct_circuits``` and ```mitiq.xyz.combine_results```.  The example below demonstrates this using Layerwise Richardson Extrapolation (LRE) applied to a Greenberger-Horne-Zeilinger (GHZ) circuit.
```

Comment:
```suggestion
This can be done by using mitiq's two stage application of `mitiq.xyz.construct_circuits` and `mitiq.xyz.combine_results`.
The example below demonstrates this using Layerwise Richardson Extrapolation (LRE) applied to a Greenberger-Horne-Zeilinger (GHZ) circuit, but the technique and particular circuit are not important.
```